.travis.yml

```diff
@@ -40,3 +40,8 @@ notifications:
 branches:
   except:
     - /^*-v[0-9]/
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then export REPO=mayadataio/dmaas-operator-amd64;
+     docker login -u "$DOCKER_USER" -p "$DOCKER_PASS" && docker build -t $REPO:ci . && docker push $REPO;
```

Comment:
To build image you can use `make image REGISTRY=maydataio` and to deploy `make deploy-image`.
we can simplify this to,
`      docker login -u "$DOCKER_USER" -p "$DOCKER_PASS" && make deploy-image`

Reply:
I don't think I got the comment. Are we looking for:
one option for make is to make the image
and another option is to push the image
If so, instead of deploy-image, can that be push-image